src/context/spt_js/client_api.js

```diff
@@ -1344,7 +1344,20 @@ TacticServerStub = function() {
 
 
     // async functions
+    this.async_ping = function(kwargs) {
+        var callback = kwargs['cbjs_action'];
+        if (!callback) {
+            callback = kwargs['callback'];
+        }
+        var on_error = function(e) {
+            callback(e);
+        };
 
+        this._delegate("ping", arguments, kwargs, null, callback, on_error);
+        return;
+    }
+    
+    

```

Comment:
I am not sure why the callback parameter is 'cbjs_action' or 'callback', it should be renamed 'on_error_callback' . Do you agree?